uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1022.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1022 {
+  
+  /**
+   * Hide Fields For Contact Summary
+   */
+  public function upgrade_1022() {
+    $valuesRemoved = $this->up1022_getContactEditOptionValues();
+    $optionValues = $this->up1022_getActiveContactEditOptionValues();
+    
+    $activeOptionValues = array_diff($optionValues, $valuesRemoved);
+    civicrm_api3('Setting', 'create', [
+      'contact_edit_options' => $activeOptionValues,
+    ]);
+    
+    return TRUE;
+  }
+  
+  /**
+   * Retrieves active contact edit option setting value
+   *
+   * @return array
+   */
+  private function up1022_getActiveContactEditOptionValues() {
+    $result = civicrm_api3('Setting', 'get', [
+      'sequential' => 1,
+      'return' => ['contact_edit_options'],
+    ]);
+    
+    return $result['values'][0]['contact_edit_options'];
+  }
+  
+  /**
+   * Retrieves value for contact edit options to be removed
+   *
+   * @return array
+   */
+  private function up1022_getContactEditOptionValues() {
+    $values = [];
+    $names = $this->up1022_getUnusedContactEditOptionNames();
+    
+    $params = [
+      'sequential' => 1,
```

Comment:
Is `sequential` required here?

Reply:
By using sequential, i did not have to worry myself with the key in my `foreach`
```php
foreach ($optionValues['values'] as $optionValue) {
```
Like you said, i think its just preference.